index.php

```diff
@@ -0,0 +1,51 @@
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Wine</title>
+
+</head>
+<body>
+<div class="titleBar">
+    <h1>Let's make some pour decisions!</h1>
+</div>
+
+<div class="container">
+    <?php
+    $db = new PDO('mysql:host=db;dbname=wineCollection', 'root', 'password'); //initialise the db connection
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
heads up, mike says that making the new PDO can be a function (for practise, more than anything else). I haven't done it yet.